tests/launch.rs

```diff
@@ -1,13 +1,26 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use kvm_ioctls::Kvm;
+use kvm_bindings::{kvm_enable_cap, KVM_CAP_MAX_VCPUS, KVM_CAP_SPLIT_IRQCHIP};
 
 use tdx::launch::TdxVm;
 
 #[test]
 fn launch() {
     let kvm_fd = Kvm::new().unwrap();
     let tdx_vm = TdxVm::new(&kvm_fd).unwrap();
+
+    let mut cap: kvm_enable_cap = Default::default();
+    cap.cap = KVM_CAP_MAX_VCPUS;
```

Comment:
I wonder if this should be a parameter to `TdxVm::new()`, and `enable_cap` should be called within that API?

Reply:
Agreed, should be good now.